frontend/src/components/layout/team-member-panel/TeamMemberPanel.tsx

```diff
@@ -1,31 +1,45 @@
 import * as React from 'react';
 import { TeamMember } from '../../atomic/team-member/TeamMember';
+import styled from '@emotion/styled';
+
+const Container = styled.div`
+  text-align: center;
+`;
+
+const Members = styled.div`
+  display: flex;
+`;
+const H2 = styled.h2`
+  height: 5vh;
+`;
 
 export const TeamMemberPanel: React.FC = (): JSX.Element => {
```

Comment:
Al tiparlo como React.FC no necesitas tipar el result ya que lo infiere TS:

```suggestion
export const TeamMemberPanel: React.FC = () => {
```